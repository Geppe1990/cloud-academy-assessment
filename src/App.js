import React from "react";
import { hot } from "react-hot-loader/root";

class App extends React.Component {
  render() {
    return (
      <>
        <h1>Title</h1>
      </>
    );
  }
}

export default hot(App);
