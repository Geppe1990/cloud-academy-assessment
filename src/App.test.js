import React from "react";
import App from "./App";

it("Renders without crashing", () => {
	shallow(<App />);
});

it('match the snapshot', () => {
const wrapper = renderer.create(<App />).toJSON();
	expect(wrapper).toMatchSnapshot();
});