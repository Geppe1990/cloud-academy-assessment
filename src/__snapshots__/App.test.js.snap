// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match the snapshot 1`] = `
<div
  id="home"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-12 text-center"
      >
        <h1>
          Rick & Morty
        </h1>
        <a
          className="btn btn-primary"
          href="/1"
          onClick={[Function]}
        >
          Go to the first character
        </a>
      </div>
    </div>
  </div>
</div>
`;
